SerialPrograms/Source/Pokemon/Options/Pokemon_StatsResetFilter.cpp

```diff
@@ -0,0 +1,144 @@
+/*  Stats Reset Filter
+ *
+ *  From: https://github.com/PokemonAutomation/Arduino-Source
+ *
+ */
+
+#include "Common/Compiler.h"
+// #include "PokemonBDSP/Inference/BoxSystem/PokemonBDSP_BoxGenderDetector.h"
+#include "Pokemon_StatsResetFilter.h"
+#include "Pokemon_EggHatchFilter.h"
+
+//#include <iostream>
+//using std::cout;
+//using std::endl;
+
+namespace PokemonAutomation{
+namespace Pokemon{
+
+const EnumDatabase<EggHatchAction>& StatsResetAction_Database() {
+    static const EnumDatabase<EggHatchAction> database({
+        {EggHatchAction::StopProgram,   "stop",     "Stop Program"},
```

Comment:
What's the difference between "keep" and "stop" for stats reset?

(I know you copy-pasted this from the egg filters, but the idea is to change places like this.)